Tools/jit/_schema.py

```diff
@@ -89,6 +92,10 @@ class COFFSection(typing.TypedDict):
     SectionData: typing.NotRequired[dict[typing.Literal["Bytes"], list[int]]]
     Symbols: list[dict[typing.Literal["Symbol"], _COFFSymbol]]
 
+class _ElfSectionType(typing.TypedDict):
+    """An ELF object file section type."""
+    Name: str
+    Value: int
```

Comment:
Are we actually using the `Value` field here? If not, we can remove this and just use `dict[typing.Literal["Name"], str]` inline below.